### DISCUSSION TITLE
skywalking-ui excludingKeywordsOfContent&keywordsOfContent not set value

### DISCUSSION THREAD

**Discussion by sunyixinyaqing** [apache/skywalking#11244] [Q&A]
<img width="1414" alt="image" src="https://github.com/apache/skywalking/assets/11975677/57b9168e-954e-41be-ad96-c8919332112b">
I entered the content keyword field when searching the log page, but these two fields in the http request have no value at all. The field name is excludingKeywordsOfContent&keywordsOfContent, and the docker image I use is skywalking-ui:8.7.0. how to solve this problem

**Comment by wu-sheng** [comment]
SkyWalking 8.x is EOL one year ago, no further update about that.

https://skywalking.apache.org/events/deprecate-v8/
